utils/docker/Dockerfile.focal

```diff
@@ -4,6 +4,8 @@ ARG DEBIAN_FRONTEND=noninteractive
 ARG TZ=America/Los_Angeles
 ARG DOCKER_IMAGE_NAME_TEMPLATE="mcr.microsoft.com/playwright:v%version%-focal"
 
+ENV LANG=C.UTF-8
```

Comment:
Other docker files also have `LC_ALL=C.UTF-8`. Not sure whether that's relevant.